rpcserverhandlers_test.go

```diff
@@ -3731,6 +3748,44 @@ func TestHandleSubmitBlock(t *testing.T) {
 	}})
 }
 
+func TestHandleVerifyChain(t *testing.T) {
+	t.Parallel()
+
+	hash := mustParseHash("00000000000000000c07ff0178ad600db22c18f8f47fa423fa144b2ca919475d")
+	depth := int64(50)
+	level := int64(1)
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleVerifyChain: ok",
+		handler: handleVerifyChain,
+		cmd:     &types.VerifyChainCmd{},
+		result:  true,
+	}, {
+		name:    "handleVerifyChain: ok with checklevel=1",
+		handler: handleVerifyChain,
+		cmd: &types.VerifyChainCmd{
+			CheckLevel: &level,
```

Comment:
I don't have any issue leaving it as-is, but if you defined the `depth` and `level` vars just so that you could pass a pointer, you could use `dcrjson.Int64` instead to do it inline